app/pages/preview/manifest.json

```diff
@@ -7,13 +7,92 @@
   "callOffAndSupplierTable": {
     "columnInfo": [
       {
-        "data": "Call-off Ordering Party"
+        "data": "Call-off Ordering Party",
+        "width": "50%"
       },
       {
-        "data": "Supplier"
+        "data": "Supplier",
+        "width": "50%"
       }
     ],
-    "columnClass": "nhsuk-grid-column-one-half nhsuk-u-font-size-16"
+    "columnClass": "nhsuk-u-font-size-16"
   },
-  "commencementDateLabel": "Commencement date of the Call-off Agreement:"
+  "commencementDateLabel": "Commencement date of the Call-off Agreement:",
+  "oneOffCostHeading": "Order items (one-off cost)",
+  "oneOffCostDescription": "The following items carry a one-off cost for the duration of this Call-off Agreement. They're the Associated Services in this order you can use as and when required",
+  "oneOffCostTable": {
+    "columnInfo": [
+      {
+        "data": "Recipient name (ODS code)",
+        "width": "20%",
+        "classes": "nhsuk-u-font-size-14"
```

Comment:
could the column class be used instead of having the fonts duplicated

Reply:
that would be nice just to set this size once. But unfortunately that doesn't work. This class needs to be set on each on each cell. I think perhaps the default nhs styling on these cells then override the one set at the top level 🤷

Reply:
ah right ok that's fair